blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -225,7 +225,7 @@ export class RunDetailsPipeline extends Component {
      */
     isParallel(nodeInfo) {
         // in case we have edges arrays we compare the first edge, if not we know we are not in parallel mode
-        return this.mergedConfig.nodeReducer.edges[0] && nodeInfo.edges[0] ?
+        return this.mergedConfig.nodeReducer.edges && this.mergedConfig.nodeReducer.edges[0] && nodeInfo.edges[0] ?
```

Comment:
`this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` `this.mergedConfig.nodeReducer.edges` 

My eyes are hurting 😜